spec/amber/cli/commands/generator_spec.cr

```diff
@@ -6,32 +6,39 @@ include CLIHelper
 include CLIFixtures
 
 module Amber::CLI
-  begin
-    describe MainCommand::Generate do
-      context "scaffold" do
-        it "generates and compile generated app" do
-          ENV["AMBER_ENV"] = "test"
-          scaffold_app(TESTING_APP)
-          MainCommand.run ["generate", "scaffold", "Animal", "name:string"]
-          assert_app_compiled?(TEST_APP_NAME)
-          cleanup
-        end
+  describe MainCommand::Generate do
+    ENV["AMBER_ENV"] = "test"
+
+    context "when scaffolding an Amber app" do
+      scaffold_app(TESTING_APP)
+      MainCommand.run ["generate", "scaffold", "Animal", "name:string"]
+      `shards build`
+
+      it "generates and compile generated app" do
+        File.exists?("bin/#{TEST_APP_NAME}").should be_true
       end
 
-      context "controllers" do
-        it "generates controller with correct verbs and actions" do
-          ENV["AMBER_ENV"] = "test"
-          controller = "Animal"
-          options = %w(add:post list:get remove:delete)
-          scaffold_app(TESTING_APP)
+      cleanup
+    end
+
+    context "when generating controllers" do
```

Comment:
Semantically, I would say we are in the context of "scaffolding and amber app" and this should be `describe "amber generate controller"`